Source/Libraries/GSF.Web/Security/AuthenticationOptions.cs

```diff
@@ -65,6 +65,11 @@ public sealed class AuthenticationOptions : Microsoft.Owin.Security.Authenticati
         /// </summary>
         public const string DefaultLoginPage = Resources.Root + "/Security/Views/Login.cshtml";
 
+        /// <summary>
+        /// Default value for <see cref="LogoutPage"/>.
+        /// </summary>
+        public const string DefaultLogoutPage = "/Security/logout";
```

Comment:
This is likely fine or intentional, but why does `DefaultLoginPage` include a `Resources.Root` prefix but `DefaultLogutPage` does not?

Reply:
The login page is a cshtml page that exists as a `GSF.Web` resource. The logout page is just hardcoded into the `AuthenticationHandler`.